src/Search.jsx

```diff
@@ -0,0 +1,136 @@
+import { useFetch } from "./hooks/useFetch";
+import { useContext, useEffect, useState } from "react";
+import { Link, useParams } from "react-router-dom";
+import { ThemeContext } from "./ThemeContext";
+import { titleCase } from "./functions/titleCase";
+
+export const Search = () => {
+	const { theme } = useContext(ThemeContext);
+	const [searchInput, setSearchInput] = useState("");
+	const { data, loading, error } = useFetch(
+		"https://pokeapi.co/api/v2/pokedex/original-unova/"
+	);
+
+	const pokemon = data.pokemon_entries;
+
+	const searchBoxChange = (event) => {
+		const val = event.target.value.toLowerCase();
+		setSearchInput(val);
+	};
+
+	const displayResults = (list) => {
+		return list.map((entry) => {
+			return (
+				<div
+					className="box"
+					style={{
+						backgroundColor: theme.boxbg,
+						border: theme.border,
+						textAlign: "center",
+					}}
+					key={entry.id}
+				>
+					<Link
```

Comment:
You can add the sprite image here by hardcoding the base URL and then dynamically adding the ID that we already have

```
  <img
      src={`https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/${entry.id}.png`}
      width={50}
   />
```